packages/browser/src/tracing/request.ts

```diff
@@ -357,6 +357,8 @@ export function xhrCallback(
     return undefined;
   }
 
+  const requestBody = JSON.parse(sentryXhrData.body as string);
```

Comment:
This is dangerous, any such operations we have to try-catch as bodies may not be JSON!